modules/account-lib/src/coin/baseCoin/iface.ts

```diff
@@ -40,17 +50,30 @@ export function isSeed(source: KeyPairOptions): source is Seed {
 /**
  * @param source
  */
-export function isPrivateKey(source: KeyPairOptions): source is PrivateKey {
+export function isPrivateKey(source: KeyPairOptions | BlsKeyPairOptions): source is PrivateKey {
```

Comment:
can we merge blsKeyPairOptions into KeyPairOptions? in order to minimize the modification to the typing here?